src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/DepthPerSampleHC.java

```diff
@@ -42,6 +43,7 @@
 @DocumentedFeature(groupName=HelpConstants.DOC_CAT_ANNOTATORS, groupSummary=HelpConstants.DOC_CAT_ANNOTATORS_SUMMARY, summary="Depth of informative coverage for each sample (DP)")
 public final class  DepthPerSampleHC implements GenotypeAnnotation, StandardHCAnnotation, StandardMutectAnnotation {
     private final static Logger logger = LogManager.getLogger(DepthPerSampleHC.class);
```

Comment:
Do we still need the normal logger?

Reply:
@lbergelson Yes, we do need to keep the regular logger for the other log messages output in these annotation classes -- we only want the "one-shot" logger to apply to the specific warning that was being spammed.